src/main/java/org/jboss/tools/intellij/openshift/utils/odo/OdoCli.java

```diff
@@ -478,9 +478,13 @@ public void logout() throws IOException {
     }
 
     @Override
-    public List<ServiceInstance> getServices(String project, String application) {
-        ServiceCatalogClient sc = client.adapt(ServiceCatalogClient.class);
-        return sc.serviceInstances().inNamespace(project).withLabelSelector(new LabelSelectorBuilder().addToMatchLabels(KubernetesLabels.APP_LABEL, application).build()).list().getItems();
+    public List<ServiceInstance> getServices(String project, String application) throws IOException {
+        try {
+            ServiceCatalogClient sc = client.adapt(ServiceCatalogClient.class);
+            return sc.serviceInstances().inNamespace(project).withLabelSelector(new LabelSelectorBuilder().addToMatchLabels(KubernetesLabels.APP_LABEL, application).build()).list().getItems();
+        } catch (KubernetesClientException e) {
+            throw new IOException("Error when getting services list: "+e.getLocalizedMessage(), e);
+        }
```

Comment:
@jeffmaury this new implementation breaks tree node UI in https://github.com/redhat-developer/intellij-openshift-connector/blob/00a7b83a7eb03993b5d233313bf91cf071a811b0/src/main/java/org/jboss/tools/intellij/openshift/tree/application/ApplicationNode.java#L40

Reply:
reverted and return empty list. in latest commit.